### PR TITLE
MNT: Re-rendered with conda-smithy 2.0.0

### PR DESCRIPTION
**Pull request by jochym**

### CODE REVIEW

**Comment by jochym** [comment]
@jakirkham or @brandon-rhodes would you merge this re-rendering? Skyfield feedstock have jplephem as dependency and I cannot release it for py3.6

**Comment by jakirkham** [comment]
Sure. Though I'd add that if this dependency is very important to you, it might be worthwhile to add yourself as a maintainer so that you can address this stuff more directly.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
     - 0001_use-setuptools.patch
 
 build:
-  number: 0
+  number: 2
```
Comment (jakirkham) [inline recipe/meta.yaml]: Why the build number bump?

Comment (jakirkham) [reply recipe/meta.yaml]: Not that we need it, but why did you chose to bump to `2` instead of `1`?

Comment (jochym) [reply recipe/meta.yaml]: Both corrected - bump by 1 and added myself to maintainers.

Comment (jakirkham) [reply recipe/meta.yaml]: Sorry, what I meant to say is I don't think we need a build number bump. Though if we did, it should increase by 1. Could you please expand on why you think a build number bump is necessary here?

Comment (jochym) [reply recipe/meta.yaml]: I was of the impression that if you rerender without version change you are supposed to increase the build number. A jump by 2 was just a mistake. I have corrected it in the 43c03a2.

**Comment by jochym** [comment]
How about now @jakirkham ?

**Comment by jochym** [comment]
@jakirkham is there anything still missing with this recipe preventing the merge, or you just busy?

**Comment by jakirkham** [comment]
Looks fine. Was kind of hoping @brandon-rhodes and/or @Juanlu001 would weigh in given it is their feedstock. 😉

**Comment by jochym** [comment]
Unfortunatly @brandon-rhodes is probably too busy - ha was warning me about it when I included him as a maintainer in skyfield feedstock. Maybe @Juanlu001 can pick up the mantle?

**Comment by brandon-rhodes** [comment]
I also lack any context on this particular diff's changes, so I can't meaningfully review the changed lines of code. So, yes, please proceed without me! :)

**Comment by jochym** [comment]
These are just re-rendering of the recipe to update the build setup and enable build for python 3.6.

**Comment by astrojuanlu** [comment]
Oops! Missed this one, merging right away :) Thanks @jochym!

**Comment by jakirkham** [comment]
Thanks @Juanlu001.

**Comment by jochym** [comment]
Thanks @Juanlu001

**Pull request merged**